core-services/prow/02_config/_plugins.yaml

```diff
@@ -332,6 +332,9 @@ config_updater:
       - vsphere
       namespaces:
       - ocp
+    build_farm_ocp_aarch64:
```

Comment:
IMHO the name is not right. `arm` is a different architecture

Reply:
Do you mean you prefer to call it arm64 rather than aarch64?

Reply:
yeah aarch64 and arm64 are same. We have arm64 used everywhere.